client/App.jsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
-
+import '@radix-ui/themes/styles.css';
+import Footer from "./components/layout/main-layout/footer/Footer";
 export default function App() {
   return (
-    <h1>Linktaaaaaaaa</h1>
+    <>
+      <h1>Linktaaaaaaaa</h1>
+      <div id="footer">
+        <Footer />
```

Comment:
You should put the h1 and div inside the Footer component, keep App.jsx clean with component imports

typically you root file/layout files follow

```ts
const Layout = () => {
  return (
    <div>
      <Header />
      <MainContent />
      <Footer />
    </div>
  );
};
```

Reply:
Thank you for the help, Daniel. 
I've removed my footer part from the App.jsx.